Examples/Makefile.in

```diff
@@ -1754,8 +1765,8 @@ go: $(SRCDIR_SRCS)
 	if test -f $(SRCDIR)$(RUNME).go; then \
 	  $(GO) $(GOCOMPILEARG) $(SRCDIR)$(RUNME).go; \
 	  if $(GOGCC) ; then \
-	    $(COMPILETOOL) $(GO) -o $(RUNME) $(RUNME).@OBJEXT@ $(GOGCCOBJS) $(OBJS) $(IOBJS); \
-	  elif $(GO12) || $(GO13); then \
+	    $(COMPILETOOL) $(GO) -o $(RUNME) $(RUNME).%@OBJEXT@ $(GOGCCOBJS) $(OBJS) $(IOBJS); \
```

Comment:
> Do you want me to rewrite that commit with the correct file or just add a
> new commit to fix the bug into the pull request?

Sorry, I don't really understand that question (I'm very new to git and github).  I would like it to be the case that I can click on the "merge pull request" button and have the right thing appear in the swig repository.